src/nvim/CMakeLists.txt

```diff
@@ -326,6 +307,34 @@ if(WIN32)
       -DDST=${PROJECT_BINARY_DIR}/windows_runtime_deps
       -P ${PROJECT_SOURCE_DIR}/cmake/WindowsDllCopy.cmake)
   add_dependencies(nvim_runtime_deps nvim_dll_deps)
+
+  add_custom_target(external_blobs
+    COMMAND ${CMAKE_COMMAND} -E make_directory ${PROJECT_BINARY_DIR}/windows_runtime_deps/platforms
+
+    COMMAND ${CMAKE_COMMAND} -E copy "${DEPS_PREFIX}/bin/ca-bundle.crt"       ${PROJECT_BINARY_DIR}/windows_runtime_deps/
```

Comment:
@equalsraf Here's what I'm doing instead of the `find_program()` + `WindowsDllCopy.cmake` approach (notice the deleted lines just above). It's a bit inelegant: because `find_program()` is not used, this is referencing `${DEPS_PREFIX}/bin/` which means this target has implicit knowledge about the `third-party/CMakeLists.txt` project.

But I don't know a better way. `WindowsDllCopy.cmake` was running `dumpbin` on the `.exe` files, which seems  useless (and wrong) since they have nothing to do with the Neovim build, we only want these `.exe` files for inclusion with the end-user package.